tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -57,7 +168,9 @@ def test_compute_default_label_features():
         Message(data={TEXT: "test d"}),
     ]
 
-    output = DIETClassifier._compute_default_label_features(label_features)
+    output = DIETClassifierGraphComponent._compute_default_label_features(
```

Comment:
how about doing the import hack `import `DIETClassifierGraphComponent` as `DIETClassifier` to avoid unnecessary diffs now and in the future?